packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -596,7 +596,7 @@ String _generateLookupBody(
       switchClauseDeferredLoadingTemplate : switchClauseTemplate)
       .replaceAll('@(localeClass)', '$className${locale.camelCase()}')
       .replaceAll('@(appClass)', className)
-      .replaceAll('@(library)', '${fileName}_${locale.languageCode}');
+      .replaceAll('@(library)', '${fileName}_${locale.toString()}');
```

Comment:
@HansMuller 
Fixed this where import was only generated with language code, it should be full locale.

Reply:
The `toString()` shouldn't be needed here

Reply:
Thanks, fixed